### ISSUE TITLE
[Epic] Voice Messages - Phase 2

### ISSUE THREAD

**Issue by jonnyandrew** [vector-im/element-meta#2081]
After phase 2, voice messages are production ready with a polished UX/UI. Any tech debt and minor bugs that accumulate can be added here.

- Depends on #2057 

```[tasklist]
### User Stories
- [ ] https://github.com/vector-im/element-meta/issues/2110
- [ ] https://github.com/vector-im/element-meta/issues/2111
- [x] [Task] Enable voice messages by default (once above are complete)
- [x] [Story] Voice messages do not play at the same time
- [ ] https://github.com/vector-im/element-meta/issues/2175
- [ ] https://github.com/vector-im/element-meta/issues/2112
- [ ] https://github.com/vector-im/element-meta/issues/2114
- [ ] https://github.com/vector-im/element-meta/issues/2113
- [ ] https://github.com/vector-im/element-meta/issues/2166
```

```[tasklist]
### Stretch goals (out of scope)
- [ ] [Story] Switch from speaker to earpiece when holding device to ear
```

## Out of scope
- Edit a voice message
- Share voice messages to other apps
- Save as draft
- Encrypting audio files stored on the device
- Decryption progress indicator
- Waveform and scrubbing UI (stretch goal)
- Typing notifications
- Reply _with_ a voice message
- Auto-play consecutive voice messages
- Interoperability with non-element clients
- Recording length limits or warnings ([discussion](https://docs.google.com/document/d/1S_Y4Fkh7sJ7S1q5KltT1HdL3oYSJMglRl7ZbdBeQya0/edit?disco=AAAA57tDRfA))
- Record a message when text formatting options are shown

## Sign-off

```[tasklist]
### Android
- [ ] Design sign-off on completion
- [ ] QA sign-off on completion
- [ ] Product sign-off on completion
```

```[tasklist]
### iOS
- [ ] Design sign-off on completion
- [ ] QA sign-off on completion
- [ ] Product sign-off on completion
```

## References

- [Product spec](https://docs.google.com/document/d/1S_Y4Fkh7sJ7S1q5KltT1HdL3oYSJMglRl7ZbdBeQya0/edit)
- [Original product spec](https://docs.google.com/document/d/1GfQBKzqrMyEXsqdW1aL0DlbJoIIwGohrPF2O2QSstVc/edit#heading=h.b1z533gwnq8w)

[Technical lead](https://gitlab.matrix.org/new-vector/internal/-/wikis/Departments/Engineering/Engineering-Roles#project-technical-lead): @jonnyandrew 

myhours project: Element X
